firewood/src/merkle/trie_hash.rs

```diff
@@ -50,3 +46,18 @@ impl Debug for TrieHash {
         write!(f, "{}", hex::encode(self.0))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_dehydrate() {
+        let zero_hash = TrieHash([0u8; TRIE_HASH_LEN]);
+
+        let mut to = [1u8; TRIE_HASH_LEN];
+        zero_hash.dehydrate(&mut to).unwrap();
+
+        assert_eq!(&to, &zero_hash.0);
+    }
+}
```

Comment:
This test existed in `merkle.rs` before

Reply:
Seems meaningless.

Reply:
It's definitely not meaningless. It's making sure that `dehydrate` writes everything to its input parameter.